src/resources/user/auth.controller.js

```diff
@@ -24,13 +28,15 @@ export const addUserInfo = async (req, res) => {
       },
       token,
     };
+    sendEmailConfirmation(data.user, token);
```

Comment:
You have access to the user object here without going through the data object

Reply:
If I destructure this, I change the format of how you pass data to the formatResponse() for user registration. something is making the test fail though it's either  me trying to destructure the data object or trying to make the expiresIn in jwt dynamic

Reply:
you don't need to destructure, just do something like `sendEmailConfirmation(user, token);`. The user variable is in scope

Reply:
Okay yeah, let me try that

Reply:
Done